### PR TITLE
Hotkey for Guild Window

### LINKED ISSUE #724: Guild Shortcut
**Is your feature request related to a problem? Please describe.**
You forgot to add that. G to Guild.

**Describe the solution you'd like**
Add guild shortcut and in the settings in options as well.

**Additional context**

### PR DESCRIPTION
**Pull request by AisenGaming**
Should resolve #724 
Not 100% sure on the JSON generation part but tested to work with a hand edited OptionsWindow.json

### CODE REVIEW

File: Intersect.Client/Core/Controls/ControlEnum.cs [hunk 1]
```diff
@@ -64,6 +64,8 @@ public enum Control
 
         OpenAdminPanel,
 
+        OpenGuild,
```
Comment (Cheshire92) [inline Intersect.Client/Core/Controls/ControlEnum.cs:67]: Any particular reason this isn't at the end of the enum?

Comment (AisenGaming) [reply Intersect.Client/Core/Controls/ControlEnum.cs:67]: No reason at all

Comment (jcsnider) [reply Intersect.Client/Core/Controls/ControlEnum.cs:67]: Should we move it above OpenSettings so it is with the other opening of game windows (Inventory/Spells/Quests/etc)?

File: Intersect.Client/Core/Controls/Controls.cs [hunk 2]
```diff
@@ -85,6 +85,7 @@ public void ResetDefaults()
             CreateControlMap(Control.OpenDebugger, Keys.F2, Keys.None);
             CreateControlMap(Control.OpenAdminPanel, Keys.Insert, Keys.None);
             CreateControlMap(Control.ToggleGui, Keys.F11, Keys.None);
+            CreateControlMap(Control.OpenGuild, Keys.G, Keys.None);
```
Comment (jcsnider) [inline Intersect.Client/Core/Controls/Controls.cs:88]: I'm not sure if this determines the order of the controls or not but we might want this above the OpenSettings line as well?

**Review by Kamusvalenzuela — DISMISSED** [review]

File: Intersect.Client/Core/Controls/Controls.cs (hunk 2 above)
Comment (Cheshire92) [reply Intersect.Client/Core/Controls/Controls.cs]: I'm not sure either but for the sake of clarity in the source files we might as well.

File: Intersect.Client/Core/Controls/ControlEnum.cs (hunk 1 above)
Comment (Cheshire92) [reply Intersect.Client/Core/Controls/ControlEnum.cs]: Probably yeah. I don't think the config uses the enum numbering but uses the names instead so it shouldn't make a difference where it's located.

**Review by Cheshire92 — DISMISSED** [review]

**Review by jcsnider — APPROVED** [review]

**Pull request merged**